docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,163 @@
+
+#### Repositories
+
+If you're looking for an active project to contribute to, this tab can help you as it contains information such as Activity, PR Overview, PR Velocity, Spam, Contributors, and activity that has occurred over the last 30 days. This information is also a good way to connect with others on the project or where to find support.  To learn more about these features, see [Understanding Insights Data](https://docs.opensauced.pizza/maintainers/understanding-lists/#understanding-lists-contributor-data).
+
+![repositories tab](../../static/img/repos-insights.png)
+
+#### Contributors
+
+The Contributors tab allows you to view more detailed information about other contributors, including Activity, Repositories, the date of their last contribution, Time Zone, and the number of contributions. This can be a great way to find other people to collaborate with or recommend for jobs.
+
+Want to learn more about your peers?  You can select them and add them to a list.
+
+![contributors insight list](../../static/img/contributors-insights.png)
+
+There are two ways to add contributors to a list:
+
+1. Create a new list with selected contributors.
+2. Add selected contributors to an existing list.
+
+## Lists: Connecting with Contributors
+
+[Lists](../features/lists.md) feature helps you connect with other contributors or learn more about what others are up to in open source. With this feature, you can categorize, watch, and check out different groups of contributors within open source projects.
+
+### What You Can Do With Lists?
+
+There are many things you can do with this feature, but here's some ideas to get you started:
+
+- Learn about what other contributors are doing in open source.
+- Learn about new projects.
+- Make connections with other contributors who share your interests and come from other backgrounds.
+
+### Creating a New List
+
+1. Click the "Lists" tab on the Insights hub to create a new list. You will be redirected to a page where you can create your new list.
+2. Click on the "New List" button to start a new list.
+3. Give your list a name.
+4. Choose your page visibility. You can choose to make your list public or private. By default, it is set to private.
+5. Add contributors to your list by searching for their GitHub username, syncing your GitHub Team, or importing your GitHub Following.
+
+![contribution sync gif](../../static/gif/contributors-sync-list.gif)
+
+## Using Your List
+
+Once you have created your list, you will be redirected to your List Page. Here, you will see a dashboard with an overview of the contributors.
+
+There are three tabs that provide you with more information on these repositories:
+
+### Overview
+
+The Overview tab gives you a high-level view of the contributors in your list, including the total number of commits in the last 30 days and the types of contributors on your list: Active, New, and Alumni. This can help be helpful for finding people to collaborate or individuals to learn from.
```

Comment:
```suggestion
The Overview tab gives you a high-level view of the contributors in your list, including the total number of commits in the last 30 days and the types of contributors on your list: Active, New, and Alumni. This can be helpful for finding people to collaborate with or individuals to learn from.
```